fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyzeStmt.java

```diff
@@ -73,144 +70,46 @@ public class AnalyzeStmt extends DdlStmt {
-        Preconditions.checkArgument(isAnalyzed(),
-                "The partitionNames must be obtained after the parsing is complete");
-        return partitionNames;
-    }
-
-    /**
-     * The statistics task obtains partitions and then collects partition statistics,
-     * we need to filter out partitions that do not have data.
-     *
-     * @return map of tableId and partitionName
-     * @throws AnalysisException not analyzed
-     */
-    public Map<Long, List<String>> getTableIdToPartitionName() throws AnalysisException {
-        Preconditions.checkArgument(isAnalyzed(),
-                "The partitionIds must be obtained after the parsing is complete");
-        Map<Long, List<String>> tableIdToPartitionName = Maps.newHashMap();
-
-        for (Table table : getTables()) {
-            table.readLock();
-            try {
-                OlapTable olapTable = (OlapTable) table;
-                List<String> partitionNames = getPartitionNames();
-                List<String> newPartitionNames = new ArrayList<>(partitionNames);
-                if (newPartitionNames.isEmpty() && olapTable.isPartitioned()) {
-                    newPartitionNames.addAll(olapTable.getPartitionNames());
-                }
-                tableIdToPartitionName.put(table.getId(), newPartitionNames);
-            } finally {
-                table.readUnlock();
-            }
-        }
-        return tableIdToPartitionName;
-    }
-
-    public Map<Long, List<String>> getTableIdToColumnName() throws AnalysisException {
-        Preconditions.checkArgument(isAnalyzed(),
-                "The db name must be obtained after the parsing is complete");
-        Map<Long, List<String>> tableIdToColumnName = Maps.newHashMap();
-        List<Table> tables = getTables();
-        if (optColumnNames == null || optColumnNames.isEmpty()) {
-            for (Table table : tables) {
-                table.readLock();
-                try {
-                    long tblId = table.getId();
-                    List<Column> baseSchema = table.getBaseSchema();
-                    List<String> colNames = Lists.newArrayList();
-                    baseSchema.stream().map(Column::getName).forEach(colNames::add);
-                    tableIdToColumnName.put(tblId, colNames);
-                } finally {
-                    table.readUnlock();
-                }
-            }
-        } else {
-            for (Long tblId : tblIds) {
-                tableIdToColumnName.put(tblId, optColumnNames);
-            }
-        }
-
-        return tableIdToColumnName;
-    }
-
-    public Map<String, String> getProperties() {
-        return optProperties;
-    }
-
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
         super.analyze(analyzer);
 
         // step1: analyze db, table and column
-        if (optTableName != null) {
-            optTableName.analyze(analyzer);
+        if (tableName != null) {
```

Comment:
if tableName must not null. we should add a check in constructor and remove this if statement